pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -138,6 +154,26 @@ func (h *NetworkUtilsHandler) IptablesAppendRule(table, chain string, rulespec .
 
 	return iptablesObject.Append(table, chain, rulespec...)
 }
+func (h *NetworkUtilsHandler) NftablesNewChain(table, chain string) error {
+	output, err := exec.Command("nft", "add", "chain", "ip", table, chain).CombinedOutput()
```

Comment:
did you look into using this package instead of forking child processes for nft? https://github.com/google/nftables

Reply:
I try to use this package but I have some issues with it:
1. It's not recommended as production
2. It's a very complex package in my opinion for the easy task the we need here.

Reply:
alrighty, just wanted to make sure it was investigated.